### PR TITLE
📱 Scale font sizes to device

### LINKED ISSUE #972: Fix responsive size for text
Ensure that all text should be scaled on smaller devices.

### PR DESCRIPTION
**Pull request by petterhh**
- 📱 Scale font sizes to device
- 🔥 Remove unused styles `labelSmall` and `desktopLinkBig`

### Reviewer instructions
You are **very** welcome to go through the variables in Figma and verify that these styles actually correspond to the ones in Figma (please do 🙏 🙏 )

### CODE REVIEW

File: src/components/text/text.module.css [hunk 1]
```diff
@@ -101,31 +135,36 @@
   line-height: 128.571%;
   letter-spacing: 0.00875rem;
 
-  @media (max-width: 375px) {
+  @media (max-width: 425px) {
     font-size: 0.75px;
```
Comment (mikaelbr) [inline src/components/text/text.module.css]: What is this? Text for ants??

Screenshot 2024-12-09 at 15 33 56

Comment (mikaelbr) [reply src/components/text/text.module.css]:
```suggestion
    font-size: 0.75rem;
```

Comment (petterhh) [reply src/components/text/text.module.css]: hahaha, good catch 🐜

**Review by mikaelbr — APPROVED** [review]
Looks good! Tested preview now on different sizes and didn't find any issues (yet)

**Review by anemne — APPROVED** [review]

**Pull request merged**